awx/api/generics.py

```diff
@@ -197,6 +200,8 @@ def finalize_response(self, request, response, *args, **kwargs):
                 logger.warning(status_msg)
         response = super(APIView, self).finalize_response(request, response, *args, **kwargs)
         time_started = getattr(self, 'time_started', None)
+        response['X-API-AWX-Version'] = get_awx_version()
+        response['X-API-AWX-Product'] = 'AWX' if isinstance(get_licenser(), StubLicense) else 'Red Hat Ansible Tower'
```

Comment:
I wanted to avoid calling `validate()` to get `license_type`, because it's expensive (relatively speaking, from an *every request* perspective)